linux/switchtec.h

```diff
@@ -120,10 +120,19 @@ struct sw_event_regs {
 } __packed;
 
 enum {
-	SWITCHTEC_CFG0_RUNNING = 0x04,
-	SWITCHTEC_CFG1_RUNNING = 0x05,
-	SWITCHTEC_IMG0_RUNNING = 0x03,
-	SWITCHTEC_IMG1_RUNNING = 0x07,
+	SWITCHTEC_BL20_RUNNING = 0x01,
+	SWITCHTEC_BL21_RUNNING = 0x02,
+	SWITCHTEC_MAP1_RUNNING = 0x03,
+	SWITCHTEC_MAP0_RUNNING = 0x04,
+	SWITCHTEC_CFG0_RUNNING = 0x05,
+	SWITCHTEC_CFG1_RUNNING = 0x06,
+	SWITCHTEC_IMG0_RUNNING = 0x07,
+	SWITCHTEC_IMG1_RUNNING = 0x08,
+};
+
+struct partition_info {
```

Comment:
we can still keep the definition of this structure where it was, since the code will need to be merged with saratoga switchtec-kernel.

Reply:
remove the partition_info, reserve the original code.